src/components/testimonials/VolunteerSection/VolunteerSection.tsx

```diff
@@ -85,7 +85,7 @@ function VolunteerSection() {
           />
         </Row>
       ))}
-      <button id="volunteers-read-more-button">Read More</button>
+      {/* <button id="volunteers-read-more-button">Read More</button> */}
```

Comment:
Why do we comment this line out?

Reply:
The original code was updated to match the Figma design. For mobile, a single ReadMore button was implemented instead of three buttons, one for each volunteer. As each volunteer has their own detail page, I am unsure how the single button would work for three volunteers' pages. As a result, I have commented out the button code for now.
<img width="112" alt="Screenshot 2024-03-04 at 1 11 01 PM" src="https://github.com/1thing-org/1thing.org/assets/81730540/169db977-5f73-4865-8a5f-5dcca325843b">